deploy/New-IdentityProvider.ps1

```diff
@@ -0,0 +1,90 @@
+<#
+.Description
+This allows you to create an Identity Provider for providing tokens
+to an application - this is optional.
+#> 
+
+[CmdletBinding()]
+param(
+    # The name of the AAD Client Application
+    [Parameter(Mandatory=$false)]
+    [string] $AADClientAppName="ccf_demo_client",
+    # The name of the AAD API Application
+    [Parameter(Mandatory=$false)]
+    [string] $AADApiAppName="ccf_demo_api",
+    # Tenant where the Application Registrations are
+    [Parameter(Mandatory=$true)]
+    [string] $TenantId
+)
+
+# The Magic Strings are well known Ids
+$MSGraphAppId="00000003-0000-0000-c000-000000000000"
+# $Scope_OpenID="37f7f235-527c-4136-accd-4a02d197296e"
```

Comment:
Do we need to commit these commented lines?

Reply:
These will be used when we move Swagger from ClientCredentials flow to AuthorisationCode flow. But I have removed them for now.